src/message_interface/account_method.rs

```diff
@@ -285,6 +285,7 @@ pub enum AccountMethod {
     /// Expected response: [`Output`](crate::message_interface::Response::Output)
     PrepareOutput {
         options: OutputOptionsDto,
+        #[serde(rename = "transactionOptions")]
```

Comment:
we should stick with `rename_all = "camelCase"` when possible i think

Reply:
never mind, the entire file is like this. smells like a cleanup job later!

Reply:
This doesn't work for nested enum fields, at least not like this
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=809f184f8234fec2c6d94a65a6d2ba7e